### ISSUE TITLE
Fix deprecation warning due to invalid escape sequence in test_metalink.py

### ISSUE THREAD

**Issue by sidcha** [vmware/tdnf#150]
CI warns about this:

```
tests/test_metalink.py:51
  /build/pytests/tests/test_metalink.py:51: DeprecationWarning: invalid escape sequence \
    utils.run([ 'sed', '-i', '-e' ,'/<verification>/a \    <hash type="md5">' + md5sum + '</hash>', photon_metalink ])

tests/test_metalink.py:62
  /build/pytests/tests/test_metalink.py:62: DeprecationWarning: invalid escape sequence \
    utils.run([ 'sed', '-i', '-e' ,'/<verification>/a \    <hash type="sha1">' + sha1sum + '</hash>', photon_metalink ])

tests/test_metalink.py:72
  /build/pytests/tests/test_metalink.py:72: DeprecationWarning: invalid escape sequence \
    utils.run([ 'sed', '-i', '-e' ,'/<verification>/a \    <hash type="sha256">' + sha256sum + '</hash>', photon_metalink ])

tests/test_metalink.py:79
  /build/pytests/tests/test_metalink.py:79: DeprecationWarning: invalid escape sequence \
    utils.run([ 'sed', '-i', '-e' ,'/<verification>/a \    <hash type="md5">' + md5sum + '</hash>', photon_metalink ])

tests/test_metalink.py:84
  /build/pytests/tests/test_metalink.py:84: DeprecationWarning: invalid escape sequence \
    utils.run([ 'sed', '-i', '-e' ,'/<verification>/a \    <hash type="sha1">' + sha1sum + '</hash>', photon_metalink ])

tests/test_metalink.py:89
  /build/pytests/tests/test_metalink.py:89: DeprecationWarning: invalid escape sequence \
    utils.run([ 'sed', '-i', '-e' ,'/<verification>/a \    <hash type="sha256">' + sha256sum + '</hash>', photon_metalink ])

tests/test_metalink.py:94
  /build/pytests/tests/test_metalink.py:94: DeprecationWarning: invalid escape sequence \
    utils.run([ 'sed', '-i', '-e' ,'/<verification>/a \    <hash type="sha256">' + sha256sum + '</hash>', photon_metalink ])

tests/test_metalink.py:99
  /build/pytests/tests/test_metalink.py:99: DeprecationWarning: invalid escape sequence \
    utils.run([ 'sed', '-i', '-e' ,'/<verification>/a \    <hash type="sha1">' + sha1sum + '</hash>', photon_metalink ])
```

**Comment by sidcha** [comment]
@tapakund, can you please look into this.

**Comment by tapakund** [comment]
https://github.com/vmware/tdnf/pull/167 fixes this.

**Comment by tapakund** [comment]
Changes merged. We can close this.

**Comment by sidcha** [comment]
Sure. You can also link your commit to this issue with a "Fixes: #150" anywhere in your commit message. That way GitHub Will close this issue automatically.
